doc/encode_effort.md

```diff
@@ -18,15 +18,17 @@ The following table describes what the various effort settings do:
 
 |Effort | Modular (lossless) | VarDCT (lossy) |
 |-------|--------------------|----------------|
-| e1 | fast-lossless, fixed YCoCg RCT, fixed ClampedGradient predictor, simple palette detection, no MA tree (one context for everything), Huffman, simple rle-only lz77 | |
-| e2 | global channel palette, fixed MA tree (context based on Gradient-error), ANS, otherwise same as e1 | |
-| e3 | same as e2 but fixed Weighted predictor and fixed MA tree with context based on WP-error | only 8x8, basically XYB jpeg with ANS |
+| e1 | fast-lossless, fixed YCoCg RCT, fixed ClampedGradient predictor, simple palette detection, no MA tree (one context for everything), Huffman, simple rle-only lz77 | only 8x8, basically XYB jpeg with ANS |
+| e2 | global channel palette, fixed MA tree (context based on Gradient-error), ANS, otherwise same as e1 | same as e1 |
+| e3 | same as e2 but fixed Weighted predictor and fixed MA tree with context based on WP-error | e2 + better ANS |
 | e4 | try both ClampedGradient and Weighted predictor, learned MA tree, global palette | simple variable blocks heuristics, adaptive quantization, coefficient reordering |
 | e5 | e4 + patches, local palette / local channel palette, different local RCTs | e4 + gabor-like transform, chroma from luma |
 | e6 | e5 + more RCTs and MA tree properties | e5 + error diffusion, full variable blocks heuristics |
 | e7 | e6 + more RCTs and MA tree properties | e6 + patches (including dots) |
 | e8 | e7 + more RCTs, MA tree properties and Weighted predictor parameters | e7 + Butteraugli iterations for adaptive quantization |
 | e9 | e8 + more RCTs, MA tree properties and Weighted predictor parameters, try all predictors | e8 + more Butteraugli iterations |
-| e10 | e9 + previous-channel MA tree properties, different group dimensions, exhaustively try various e9 options | |
+| e10* | e9 + global MA tree + other whole image optimizaions | e9 + I don't know |
+| e11* | e10 + previous-channel MA tree properties, different group dimensions, exhaustively try various e10 options | N/A |
 
+__*For efforts 10 and higher, streaming encoding is disabled. This is a significant performance hit for larger images.__<br>
```

Comment:
Consider 'chunked' instead of 'streaming' to communicate the distinction with actual streaming output. IMO, it is redundant to suggest a performance hit, where true (both 'performance' and 'larger' are vague as written). Why not just mention disabled chunked encoding in the table proper?